draft-ietf-rtcweb-jsep.xml

```diff
@@ -1302,6 +1302,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <xref target="I-D.ietf-mmusic-msid"></xref>, Section 4.</t>
             </list></t>
 
+	  <t>The next step is to generate lip sync groups as defined
```

Comment:
Since the mids are not all known at this point (only after m= sections are generated), I think this needs to happen down at line 1493, next to where we create the BUNDLE group.